artcommon/artcommonlib/assembly.py

```diff
@@ -294,7 +294,14 @@ def assembly_basis_event(
     if target_assembly.basis.brew_event:
         return int(target_assembly.basis.brew_event)  # Integer for Brew event
     elif target_assembly.basis.time:
-        return target_assembly.basis.time  # Datetime for Konflux
+        # datetime UTC for Konflux
+        time_str = target_assembly.basis.time
+        if isinstance(time_str, datetime):
```

Comment:
According to the validator, this must be of  type string; shall we enforce that here too?

Reply:
I've added validation and tests